docs/docs/setting-up-ci-cd.mdx

```diff
@@ -38,20 +38,22 @@ you can make a test run only if the pull request has a certain label (e.g. “NL
 
 ### Validating Data and Stories
 
-Data validation verifies that there are no mistakes or major inconsistencies in your domain, NLU 
-data, or conversation data. To validate your data, have your CI run this command:
+Data validation verifies that no mistakes or major inconsistencies appear in your domain, NLU 
+data, or story data. To validate your data, have your CI run this command:
 
 ```bash
 rasa data validate --fail-on-warnings --max-history <max_history>
 ```
 
-If you pass a `max_history` value to a Memoization policy in your `config.yml` file, provide the 
-same value in the above validator command. Otherwise, provide the default value of `5`.
+If you pass a `max_history` value to one or more policies in your `config.yml` file, provide the 
+smallest of those values in the above validator command. Otherwise, drop the `--max-history` flag.
```

Comment:
We should rather tell them the initial command
```
rasa data validate
```
and then tell them to add the max history flag if .... 
add the fail on warnings if ...